docs/scripts/build-props-table.mjs

```diff
@@ -0,0 +1,64 @@
+import fs from 'fs-extra';
```

Comment:
This should be zx script, so you have access to all relevant libs (fs-extra, globby, etc)

Reply:
zx doesn't include fs-extra :( @sebald

Reply:
https://google.github.io/zx/api#fs :P

Reply:
aaaaaaaaaaaah mb ✅